GameData/RealismOverhaul/Engine_Configs/XLR11_Config.cfg

```diff
@@ -242,15 +272,69 @@
 				key = 1 208
 			}
 		}
+		CONFIG
+		{
+			name = XLR35-RM-1
+			minThrust = 9.39
+			maxThrust = 27.56
+			massMult = 0.8726
+			heatProduction = 100
+			%pressureFed = False
+			%ignitions = 1
+			gimbalRange = 5 // Guess
+			PROPELLANT
+			{
+				name = Ethanol75
+				ratio = 0.4855
+				DrawGauge = True
+			}
+			PROPELLANT
+			{
+				name = LqdOxygen
+				ratio = 0.5145
+			}
+
+			PROPELLANT
+			{
+				name = HTP
+				ratio = 0.01
+				ignoreForIsp = True
+				DrawGauge = False
+			}
+
+			atmosphereCurve
+			{
+				key = 0 234.4
+				key = 1 211
+			}
+		}
 	}
 }
+@PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[XLR35-RM-1]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
+{
+	TESTFLIGHT // No data except for 2 failures on 3 test flights, keeping same
+	{
+		name = XLR35-RM-1
+		ratedBurnTime = 3600		//very simple and primitive, reused extensively. Give 1 hour
+		ratedContinuousBurnTime = 360
```

Comment:
This should done with a ratedBurnTime and a testedBurnTime (like e.g. the LR79) since it has only one ignition so a continuous burn time doesn't make sense (that's for reignitable engines).